install/yamls/ccruntime-peer-pods-in-pod.yaml

```diff
@@ -1,12 +1,11 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: DaemonSet
 metadata:
-  name: cloud-api-adaptor-deployment
+  name: cloud-api-adaptor-daemonset
   namespace: confidential-containers-system
   labels:
     app: cloud-api-adaptor
 spec:
```

Comment:
Would it make sense to add a nodeSelector field to the yaml with default set to all workers ?

Reply:
yep, makes sense. Will update